src/reference/config/bindings/binding-openapi-asyncapi.md

```diff
@@ -0,0 +1,161 @@
+---
+shortTitle: openapi-asyncapi
+description: Zilla runtime openapi-asyncapi binding
+category:
+  - Binding
+tag:
+  - Proxy
+---
+
+# openapi-asyncapi Binding
+
+Zilla runtime openapi-asyncapi binding.
+
+```yaml {2}
+openapi_asyncapi_proxy:
+  type: openapi-asyncapi
+  kind: proxy
+  options:
+    specs:
+      openapi:
+        openapi-id: spec/openapi.yaml
+      asyncapi:
+        asyncapi-id: spec/asyncapi.yaml
+  routes:
+    - when:
+        - api-id: openapi-id
+      exit: asyncapi_client
+      with:
+        api-id: asyncapi-id
+```
+
+## Summary
+
+The `proxy` kind `openapi-asyncapi` binding adapts `opanapi` request-response streams to `asyncapi` streams.
```

Comment:
```suggestion
The `proxy` kind `openapi-asyncapi` binding adapts OpenAPI request-response streams to AsyncAPI streams.
```
Unless you meant to mention how this binding adapts other binding types I think it is better to use the OpenAPI and AsyncAPI names.